test/index.js

```diff
@@ -2,33 +2,88 @@ const lkl = require('../');
 const fs = require('fs');
 const path = require('path');
 const assert = require('assert');
+const async = require('async');
+const zlib = require('zlib');
 
 let mountpoint;
 
 describe('lkl', function() {
 	before(function(done) {
-		this.disk = __dirname + '/tmp.ext4';
-
+		const self = this;
+		this.disk = __dirname + '/tmp.ext4';  // ext4 partition
+		this.partitionnedDisk = __dirname + '/tmp.img';  // whole disk
+		this.partitions = [  // whole disk partitions
+			{ number: 1, fsType: 'btrfs' },
+			{ number: 2, fsType: 'ext2' },
+			{ number: 3, fsType: 'ext4' },
+			// 4 is an extended partition
+			{ number: 5, fsType: 'vfat' },  // fat16
+			{ number: 6, fsType: 'vfat' },  // fat32
+			{ number: 7, fsType: 'xfs' },
+		]
+
+		// create a copy of the ext4 partition
 		fs.createReadStream(__dirname + '/fixtures/test.ext4')
 		.pipe(fs.createWriteStream(this.disk))
-		.on('close', done);
+		.on('close', function() {
+			// then create a copy of the partitionned disk.
+			// The disk image is compressed in order to avoid github's file
+			// size restrictions.
+			fs.createReadStream(__dirname + '/fixtures/disk.img.gz')
```

Comment:
The uncompressed size is 343MB. Why not something much smaller like the 4MB ext4 partition?